applications/StructuralMechanicsApplication/tests/cpp_tests/test_shells_matrices.cpp

```diff
@@ -128,7 +128,21 @@ void ConductShellMassMatrixTest(std::string const& rElementName, const Matrix& r
 
     Matrix lhs;
     p_elem->CalculateMassMatrix(lhs, r_process_info);
-    KRATOS_CHECK_MATRIX_NEAR(lhs, rRefMatrix, 1e-12);
+
+    // for writing the reference values
```

Comment:
Leftover?

Reply:
I want to leave it there in case one needs to rewrite the reference values in the future
(half of the time making the test I spent on this :D)

Reply:
Should be then be a c++ test instead?

Reply:
What do you mean?
This is a cpp test

Reply:
Ups